### PR TITLE
Use const in configmap test, eliminate stale comments

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Cleanup

/assign @jhadvig

### CODE REVIEW

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -96,7 +84,7 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		kubeClient,
 		resync,
 		informers.WithNamespace(api.TargetNamespace),
-		informers.WithTweakListOptions(tweakListOptions),
+		informers.WithTweakListOptions(tweakListOptionsNOOP),
```
Comment (spadgett) [inline pkg/console/starter/starter.go]: Do we even need to call this if we're not setting an options?

Comment (benjaminapetersen) [reply pkg/console/starter/starter.go]: We can probably just eliminate it at this point.

**Comment by jhadvig** [comment]
```
fail [k8s.io/kubernetes/test/e2e/kubectl/kubectl.go:501]: Expected error:
    <exec.CodeExitError>: {
        Err: {
            s: "error running &{/usr/bin/kubectl [kubectl --server=https://api.ci-op-nms7ktkn-ab436.origin-ci-int-aws.dev.rhcloud.com:6443 --kubeconfig=/tmp/admin.kubeconfig --namespace=e2e-tests-kubectl-c287d exec nginx -- /bin/sh -c exit 0] []  <nil>  Unable to connect to the server: net/http: TLS handshake timeout\n [] <nil> 0xc42328d320 exit status 1 <nil> <nil> true [0xc422e6e350 0xc422e6e368 0xc422e6e380] [0xc422e6e350 0xc422e6e368 0xc422e6e380] [0xc422e6e360 0xc422e6e378] [0x8b8bb0 0x8b8bb0] 0xc422577620 <nil>}:\nCommand stdout:\n\nstderr:\nUnable to connect to the server: net/http: TLS handshake timeout\n\nerror:\nexit status 1\n",
        },
        Code: 1,
    }
...
````

/retest

**Pull request merged**